restapi/src/main/java/io/stargate/web/resources/RequestHandler.java

```diff
@@ -45,6 +46,13 @@ public static Response handle(Callable<Response> action) {
                   "Resource not found: " + nfe.getMessage(),
                   Response.Status.NOT_FOUND.getStatusCode()))
           .build();
+    } catch (OverloadedException e) {
+      logger.info("Overloaded", e);
```

Comment:
Duplicated to mimic the existing code, but logging errors at info feels weird.

Reply:
True.. Just imagine the size of the log once we are hitting the `OverloadedException`.. Let's log only the important stuff, something we don't know why is it happening, or?

Reply:
You're right, this is definitely a bad idea, removing it.